openprescribing/matrixstore/build/import_prescribing.py

```diff
@@ -139,7 +139,7 @@ def build_matrices(prescriptions, practices, dates):
     grouped_by_bnf_code = groupby(prescriptions, lambda row: row[0])
     for bnf_code, row_group in grouped_by_bnf_code:
         items_matrix = sparse_matrix(shape, integer=True)
-        quantity_matrix = sparse_matrix(shape, integer=True)
+        quantity_matrix = sparse_matrix(shape, integer=False)
```

Comment:
So we've talked about keeping this an integer but multiplying by 1000. Now's not the time but we should keep that in mind.